pageserver/src/pgdatadir_mapping.rs

```diff
@@ -51,7 +51,7 @@ pub enum LsnForTimestamp {
 /// versions, if there are big changes in how the data is organized in the data
 /// directory, or if new special files are introduced.
 ///
-pub trait DatadirTimeline: Timeline {
```

Comment:
Comment above needs to be adjusted, it says "This trait ..." which is no longer true